code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -52,89 +75,178 @@
+    - name: Application Profiles
+      description: Operations to define, read and manage an application's thresholds for network quality (latency, jitter, loss, throughput)
+    - name: Notifications Management
+      description: Create and manage a subscription to receive periodic connectivity insights
    
    
     paths:
-      /network-monitoring/policy:
+      /network-info/insights:
+        get:
+          security:
+          - openId:
+            - connectivity_insights:network_info:read
+          tags:
+          - Connectivity Insights
+          description: Determine the type of network the given device is connected to, and if it is a cellular network, the network identity and connectivity type (3G/4G/5G-SA/5G-NSA)
+          operationId: readNetworkInfoInsight
+          responses:
+            "200":
+              description: OK
+              content:
+                application/json:
+                  schema:
+                      $ref: '#/components/schemas/NetworkInfoInsight'
+                      
+      /network-quality/insights:
         post:
           security:
-          - openId: 
-            - connectivity_insights:policy:write
+          - openId:
+            - connectivity_insights:network_quality_insight:write
           tags:
-          - Network Monitoring Policy Management
-          summary: create a policy which represents an application's networking demands. 
-          description: Define network monitoring intents for optimal end user application experience
-          operationId: createPolicy
+          - Quality Insights
+          description: Create an insight. The response shows the network's current level of confidence that it can meet an application profile's quality thresholds for a given end user device. An `insightId` is included to support polling via the GET operation, or to requrest callbacks via the Notification Management operations.
+          operationId: createNetworkQualityInsight
           requestBody:
-            description: List of user object
+            description: An `ApplicationProfileId` and one or more device identifiers. Together these allow the network to calculate whether the thresholds defined in the application profile can be met for the particular connected device.
             content:
               '*/*':
                 schema:
-                  $ref: '#/components/schemas/NetworkMonitoringPolicy'
+                  $ref: '#/components/schemas/NetworkQualityInsightRequest'
             required: true
           responses:
             "200":
-              description: good
+              description: OK
+              content:
+                application/json:
+                  schema:
+                      $ref: '#/components/schemas/NetworkQualityInsight'
+                      
+      /network-quality/insights/{insightId}:
```

Comment:
there is no subscription being created here. there is no role for a GET and DELETE here.